### ISSUE TITLE
REVDEP matrixStats: formals(dms_generic) not identical to formals(ms_fun)

### ISSUE THREAD

**Issue by HenrikBengtsson** [PeteHaitch/DelayedMatrixStats#39]
I'm [observing](https://github.com/HenrikBengtsson/matrixStats/blob/59c111a29c53fda83d756a1a427d608e912086a5/revdep/problems.md#delayedmatrixstats) the following error when I run revdep checks on the `develop` branch of matrixStats:
```r
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Component 8: target is NULL, current is name
  
  ── 2. Failure: Formals of S4 generic matches matrixStats function (@test_S4_gene
  formals(dms_generic) not identical to formals(ms_fun).
  Names: 1 string mismatch
  Length mismatch: comparison on first 8 components
  Component 8: target is NULL, current is name
  
  ══ testthat results  ═══════════════════════════════════════════════════════════
  OK: 8837 SKIPPED: 2 FAILED: 2
  1. Failure: Formals of S4 generic matches matrixStats function (@test_S4_generics.R#10) 
  2. Failure: Formals of S4 generic matches matrixStats function (@test_S4_generics.R#10) 
  
  Error: testthat unit tests failed
  Execution halted
```

Could that be related to https://github.com/HenrikBengtsson/matrixStats/blob/develop/NEWS#L34-L35?

**Comment by PeteHaitch** [comment]
Thanks! I'll fix that once the next version of **matrixStats** is on CRAN (don't think I can do yet because my build will fail on BioC...perhaps I can make the test conditional)

Those tests are to ensure I catch any upstream changes you make to the **matrixStats** API

**Comment by HenrikBengtsson** [comment]
Ok, I just wanted to make sure it wasn't anything serious.

**Comment by PeteHaitch** [comment]
Fixed in 08909ed57fc2fcb405d66e73ca2f614cd08a5c37
